README.md

```diff
@@ -77,21 +77,23 @@ To generate a project from the template:
 
 -   Or in Python code, programmatically:
 
-    ```python
-    from copier import run_copy
+        ```python
+        from copier import run_copy
 
-    # Create a project from a local path
-    run_copy("path/to/project/template", "path/to/destination")
+        # Create a project from a local path
+        run_copy("path/to/project/template", "path/to/destination")
 
-    # Or from a Git URL.
-    run_copy("https://github.com/copier-org/copier.git", "path/to/destination")
+        # Or from a Git URL.
+        run_copy("https://github.com/copier-org/copier.git", "path/to/destination")
 
-    # You can also use "gh:" as a shortcut of "https://github.com/"
-    run_copy("gh:copier-org/copier.git", "path/to/destination")
+        # You can also use "gh:" as a shortcut of "https://github.com/"
+        run_copy("gh:copier-org/copier.git", "path/to/destination")
 
-    # Or "gl:" as a shortcut of "https://gitlab.com/"
-    run_copy("gl:copier-org/copier.git", "path/to/destination")
-    ```
+        # Or "gl:" as a shortcut of "https://gitlab.com/"
+        run_copy("gl:copier-org/copier.git", "path/to/destination")
+        ```
+
+    If `path/to/destination` doesn't exist, Copier will create the directory.
```

Comment:
I think you can just leave it documented in ˋgenerating.mdˋ. This way, the README is shorter. It's not something with so much importance so as to live in the README IMHO.